docs/advanced-topics/dockstore-cli/advanced-features.rst

```diff
@@ -330,14 +330,15 @@ with the `Cromwell <https://github.com/broadinstitute/cromwell>`__ version liste
 +-------------+-----------------------+
 |     1.11    |          57           |
 +-------------+-----------------------+
-
+|     1.12    |          77           |
++-------------+-----------------------+
 
 Additionally, you can override the cromwell version in your
 ``~/.dockstore/config`` using for example:
 
 ::
 
-    cromwell-version = 58
+    cromwell-version = 78
```

Comment:
77?

Reply:
its an example of bumping the cromwell version to a newer version than what is natively supported in Dockstore. So if there was a version 78 of Cromwell, you could use that in the CLI by overriding the Cromwell version.

Reply:
To decrease confusion, I think we should consider putting in a previous version, even though that's generally not something someone would do.

Reply:
Also: Let's say someone sets cromwell-version = 60. Would the Dockstore CLI download Cromwell 60 on the fly or would the JAR need to already exist in a certain directory? Whichever of these happens should probably be briefly documented.

Reply:
fixed